### PR TITLE
Remove DogStatsD and minor cleanup

### PR DESCRIPTION
**Pull request by srikanthccv**
Fixes https://github.com/SigNoz/engineering-pod/issues/300

### CODE REVIEW

**Comment by zriyansh** [comment]
@ankit01-oss @pranay01 

can you please look into this PR?

**Comment by pranay01** [comment]
@ankitnayan May be you want to take a look at it

File: requirements.txt [hunk 1]
```diff
@@ -7,6 +7,6 @@ sqlparse==0.3.1
 whitenoise==5.2.0
 dj-database-url==0.5.0
 datadog==0.43.0
```
Comment (ankitnayan) [inline requirements.txt]: @srikanthccv is this library needed anymore?

Comment (srikanthccv) [reply requirements.txt]: No, it's not.

Comment (srikanthccv) [reply requirements.txt]: Addressed in c6535349401a0808eb666f615cfb73be9049d3f0

**Pull request merged**